src/components/userDashboard/components/orders/components/orderDetails.jsx

```diff
@@ -0,0 +1,124 @@
+/* eslint-disable react/jsx-no-literals */
+/* eslint-disable react/jsx-fragments */
+/* eslint-disable react/display-name */
+import React from 'react';
+import { useSelector } from 'react-redux';
+import Moment from 'react-moment';
+import {
+    Card, Col, Descriptions, Row, Tag
+} from 'antd';
+import { DETAILS_STRINGS } from '../constant';
+
+const {
+    EST_TIME,
+    GARMENT,
+    NO_DETAILS,
+    PRICE,
+    QUANTITY,
+    STATUS,
+    VENDOR,
+} = DETAILS_STRINGS;
+
+function ClothItem(item) {
+    return (
+        <div key={item.key}>
+            {/* eslint-disable-next-line react/jsx-no-literals */}
+            {`${GARMENT} ${item.garmentDesc}|
+            ${item.garment}`}
+            <br />
+            {`${PRICE} ${item.garmentPrice}`}
+            <br />
+            {`${QUANTITY} ${item.quantity}`}
+            <br />
+            {`${EST_TIME} ${item.estimatedTime}`}
```

Comment:
I think you can use string concatenation here 

```suggestion
          
   {EST_TIME + item.estimatedTime}
```